spec/models/hotel_spec.rb

```diff
@@ -3,5 +3,6 @@
 RSpec.describe Hotel, type: :model do
   describe "relationships" do
     it { should have_many :rooms }
+    it { should have_many(:guests).through(:rooms) }
```

Comment:
Good job adding this test! Don't forget to add a model test file for GuestRoom as well.